drivers/gpio/gpio_nrf5.c

```diff
@@ -65,24 +65,22 @@ struct _gpiote {
 	__IO u32_t CONFIG[8];
 };
 
+/*@todo: move GPIOTE channel management to a separate module */
+static u32_t gpiote_chan_mask;
+
 /** Configuration data */
 struct gpio_nrf5_config {
+	/* GPIO port */
+	u32_t gpio_port;
```

Comment:
Is it worth spending `u32_t` to hold values 0-1?
I'd make it `u8_t` and move it to the end of the structure to avoid padding (which will probably appear anyway between structures for two ports when the linker places them one after another, but that's another story).